medusa/tv/series.py

```diff
@@ -1323,8 +1271,8 @@ def make_ep_from_file(self, filepath):
         :return:
         :rtype: Episode
         """
-        if not os.path.isfile(filepath):
-            log.info(u"{indexer_id}: That isn't even a real file dude... {filepath}",
+        if not (os.path.isfile(filepath) and helpers.is_media_file(filepath)):
```

Comment:
Don't know if the `is_media_file` check is really needed, 
since `Series.load_episodes_from_dir` only lists media files?
https://github.com/pymedusa/Medusa/blob/abdd3b6d2008a23c4069629ed68561572184289c/medusa/tv/series.py#L969

Reply:
You are right, but since want to keep separating the logic, I thought we could keep it. It is coming from the old `should_refresh_file`: https://github.com/pymedusa/Medusa/pull/4157/files#diff-01e0cabaabc8a78fb76ee292b696e98aL1282